test/libsolidity/syntaxTests/literalSuffixes/parameters/exponent_out_of_range.sol

```diff
@@ -16,13 +17,15 @@ contract C {
 
     function max() public pure {
         1e-256 u8;
+        128e-256 mi8eu8;
```

Comment:
Not sure if we should have a separate file dedicated to both mantissa and exponent being out of range, so for now I just added this case here and also in `mantissa_out_range.sol`.